docs/MinimumNodeService.md

```diff
@@ -0,0 +1,174 @@
+# Minimum Node Service Interface
+
+The Minimum Node Service provides the basic VLCB interface and functions for the library. As such,
+it is the only service whose inclusion is mandatory.
+
+The interface looks like this:
+```
+class MinimumNodeService : public Service
+{
+public:
+  virtual void setController(Controller *cntrl) override;
+  virtual void process(UserInterface::RequestedAction requestedAction) override; 
+  virtual Processed handleMessage(unsigned int opc, VlcbMessage *msg) override;
+
+  virtual byte getServiceID() override { return SERVICE_ID_MNS; }
+  virtual byte getServiceVersionID() override { return 1; }
+  
+  virtual void begin() override;
+  
+  // backdoors for testing
```

Comment:
Please take out these lines that are used for testing. You don't describe them.

Reply:
Do you mean all of the lines or just the ones regarding backdoors?

Reply:
Just the backdoor ones. Exactly as you did it.